### DISCUSSION TITLE
Do you hate selfhosters?

### DISCUSSION THREAD

**Discussion by devcomputer** [docusealco/docuseal#273] [General]
From what I see docuseal is in a dead end, e.g. check the pricing.

Selfhosted PRO costs the same as a full hosted Service with PRO.

Why the same price? Only think I can think of is greed. 
Why motivate people to selfhost and think about gdpr if you can sell subscriptions. 
Expensive subscriptions, even is you selfhost.

Want to selfhost and replace a logo? well well well, you will need to pay 20$ per user PER MONTH.

pride comes before a fall.

**Comment by devcomputer** [comment]
![image](https://github.com/docusealco/docuseal/assets/35902641/e81a85fb-8eeb-48d5-96a4-9cd4f0a7d69f)

![image](https://github.com/docusealco/docuseal/assets/35902641/460df690-3044-4aa9-ab81-1fc506e7df8e)

surely you do not want people outside of the first world to use your tool. those damn ppl that do not have money and do not want to share data to your sad cloud....

![image](https://github.com/docusealco/docuseal/assets/35902641/b63b0f05-0b8c-4877-a415-3e46937462ef)

just a terrible that even paying customers have to opt in. I mean they pay.... poor US customers, right?

**Comment by devcomputer** [comment]
how much of their earnings does docuseal give back to other opensource projects they rely on like SSL? 
how much did you transfer over to the ppl at git?

what percentage? surely should justify the 20$ per month per user steal.

**Comment by devcomputer** [comment]
ah found it...**opensign is much better.**

k thanx bye

**Comment by compgeniuses** [comment]
wouldnt it be beneficial to have an official response in regards to this?

**Reply by omohokcoj** [reply]
Hi @compgeniuses 
Open-source self-hosted is perfect for personal use and small teams, while self-hosted Pro has the features for businesses to sign documents at scale.
The self-hosted Pro price is justified by the additional expenses involved in its development and the amount of tech support it requires to serve self-hosted customers.
The Pro hosted price is the same as the Pro Cloud.

**Reply by compgeniuses** [reply]
IN my belief, the self hosted has minimal support from your end, because technically, the technical work has been outsourced to the Implementor of the Self hosted version.

Most Self hosted versions are implemented by an MSP or so, 

While the Free perspective is the premise, not all business want it for free and are willing to actually Pay to support development, and for the Case of your solution, to get additional Features.

We, at our own volution have created a Module of Docuseal for Nethserver https://github.com/geniusdynamics/ns8-docuseal

WHich is a small business suite. I would like to Pay for Pro version, and since we are an MSP, we wiould deloy and support our clients on technical matters, like migration, installation backup and so on for the same. 

The cost of server is mine, not yours, the technical work and time incurred is from my end, including the building of the module, which we did without support from your end,

IF the charge for Self hosted makes sense, its easy for me to add a markup for managing to my clients, and still pay you for development, and its win win, most law firms and accounting firms would love to just own their data.

Wouldnt a proper well structured Fee for self hosters be more reasonable, Either Per Server Licnese, or a more cheaper per user Pricing.
Considering even Ceritificates are self acquired.

**Reply by omohokcoj** [reply]
@compgeniuses unfortunately we're not planning to change our Pro self-hosted prices due to the additional development expenses and support hours involved in serving self-hosted customers. For MSP/re-sellers like you we offer volume discounts, please feel free to reach out sales@docuseal.co to learn which re-seller discount plan might fit based on your usage.

**Reply by compgeniuses** [reply]
its ok and understandable, just make it more sensible, SInce in cloud you incur server costs, in self hosted, you do not, just have that in mind

**Reply by JohnAbassian** [reply]
I'm considering using docuseal, but this is just ridiculous.

>The self-hosted Pro price is justified by the additional expenses involved in its development 

That's a fair fundraising option, but make it optional. Come on, do I really need to explain why having your logo on documents you sign is very important for most organizations' average user to not freak out and delete the email?

By hiding the logo change behind pro pricing, you've made the self hosted product useless to most orgs. This is obviously intentional.

>and the amount of tech support it requires to serve self-hosted customers.

Stop gaslighting us, we don't want support, we just want to put our logo on it so our users can trust our emails. 


If I'm choosing between implementing a full in-house maintained self hosted solution for my org, I have to pay staff to maintain it, I have to manage security and storage and backup, and I'm being asked to pay the same price as an org that simply pays you to do it all? Just to put my logo on an open source product I installed and I'm maintaining? 

This is insulting.

**Comment by ZinZingoo** [comment]
First of all why are you interested in this crapy-ware,
When i see their website, I was so excited,
but right after i installed and then tried it,i was so disappointed, 
paying for this... seriously.

**Comment by nanoscopic** [comment]
@devcomputer 

"Docuseal is a dead end because of the pricing" How so? The available agpl code works fine and does useful things.

I agree it is a little silly that branding changes aren't available as-is, but you can simply fork and alter the code. There is an available fork that has the ability to upload your own logo already as well.

"opensign is much better" How so? It doesn't provide SMTP custom-domain sending without paying for it, and docuseal does.

"it is crappy-ware" If you feel this way, simply don't use it and use something you like. Why do you feel the need to attack the project and company as you are?

**Comment by cambordas** [comment]
While I don't endorse the confrontational tone of some of the comments here, they address some underlying concerns about Docuseal's self-hosted model.

#### Pricing between On-cloud vs self-hosted

Users are confused why self-hosted Pro costs the same as on-cloud. For self-hosters, they already absorb hosting costs, maintenance and technical labor. Yet the licensing fee doesn't reflect burden on Docuseal's infrastructre

#### Feature accessibility for basic needs

Some feature likes branding and templates APIs are essential for organizations to build trust for their clients and integrate/extend their workflows. Locking these behind Pro creates friction for small teams testing viability.

----

Why this matters? Small teams and developers are often the ones advocating for tools like Docuseal within larger organizations. If the barrier to testing is too high  (e.g., $20/user/month just to brand documents), they will seek alternatives, even if Docuseal would appeal as the better software.

By refining the self-hosted model, docuseal could:
- Expand adoption
- Increase conversion to paid plans as team scales
- Strengthen its reputation

-----

Suggestion that might help both parties:

- Introduce a self-hosted-specific tier with reduced per-user fees or some kind  of "bring- your-own-infrastructure" discount. This would align pricing with value and incentivize adoption
- This variant could provide basic branding and core API features (e.g. Create templates from PDF). The Pro Self-hosted would introduce the advanced features. This could mirror other open-core models (e.g GitLab) where core features drive adoption and enterprises pay for scalability

----

I don't think this is about entitlement, but rather aligning pricing with realities of self-hosting and fostering a sustainable ecosystem. Kudos to Docuseal for the great product, I am eager to adopt it for my team, but the current status make it challenging.  I hope this feedback contributes to solutions that empower more teams to leverage Docuseal while supporting its growth
